flake.nix

```diff
@@ -11,9 +11,9 @@
   outputs = { self, nixpkgs, nixpkgsOld }:
     let
       overlay = self: super: {
-        # this overlay is to replace yarn's dep from the latest nodejs, to nodejs 16.
+        # this overlay is to replace yarn's dep from the latest nodejs, to nodejs 18.
         yarn = with self;
```

Comment:
Presumably this shouldn't be `yarn` any more? Although looking at the rest of the file, I see references to python36 that we're also not using, so maybe this flake is out of date already?